scripts/gha/github.py

```diff
@@ -155,9 +155,40 @@ def list_artifacts(token, run_id):
 
 def download_artifact(token, artifact_id, output_path):
   """https://docs.github.com/en/rest/reference/actions#download-an-artifact"""
-  url = f'{FIREBASE_URL}/actions/artifacts/{artifact_id}/zip' 
+  url = f'{FIREBASE_URL}/actions/artifacts/{artifact_id}/zip'
   headers = {'Accept': 'application/vnd.github.v3+json', 'Authorization': f'token {token}'}
   with requests.get(url, headers=headers, stream=True, timeout=TIMEOUT) as response:
     logging.info("download_artifact: %s response: %s", url, response)
     with open(output_path, 'wb') as file:
         shutil.copyfileobj(response.raw, file)
+
+
+def dismiss_review(token, pull_number, review_id, message):
+  """https://docs.github.com/en/rest/reference/pulls#dismiss-a-review-for-a-pull-request"""
+  url = f'{FIREBASE_URL}/pulls/{pull_number}/reviews/{review_id}/dismissals'
+  headers = {'Accept': 'application/vnd.github.v3+json', 'Authorization': f'token {token}'}
+  data = {'message': message}
+  with requests.put(url, headers=headers, data=json.dumps(data),
```

Comment:
I believe is safe to use `requests_retry_session()` both here and for `get_reviews `.

Reply:
I'll add this in a follow up. If we have to retry this, should it just get the current page again, or should it get all the pages from scratch?

Reply:
For `get_reviews`, it's get current page again.